docker-for-aws/index.md

```diff
@@ -7,6 +7,15 @@ redirect_from:
 - /engine/installation/amazon/
 ---
 
+{% include d4a_buttons.md %}
+
+## Quickstart
+
+If your account has the proper permissions (outlined later in this document),
```

Comment:
It's unclear what account is meant here. I also think what skip the permission stuff since this is a quickstart, and it's probably not a terrible assumption that the user's account is ok.

I recommend replacing with "Click this button to install Docker for AWS using CloudFormation"